Utils/trigger_private_build.py

```diff
@@ -123,28 +136,40 @@ def main():
 
     if branch_has_private_build_infra_change(branch_name):
         # get the workflows ids before triggering the build
-        pre_existing_workflow_ids = get_dispatch_workflows_ids(github_token, 'master')
+        pre_existing_workflow_ids = get_dispatch_workflows_ids(github_token, private_branch_name)
 
         # trigger private build
-        payload = {'event_type': f'Trigger private build from content/{branch_name}',
-                   'client_payload': {'commit_sha1': branch_name, 'is_infra_build': 'True'}}
-
-        res = requests.post(TRIGGER_BUILD_URL,
+        if private_branch_name == 'master':
```

Comment:
Is there a shared way to pass the parameters here so it will be on the same fields? 
Now the case is client_payload for dispatch on master, and inputs for the custom private branch. 
This makes a messy check in the private build instead of using the same convention

Reply:
the only solution for that is to move to inputs only.
but this will prevent us from giving a title for the build which might be a little annoying since you can’t search it.